modules/data-entry/src/main/frontend/src/questionnaire/Question.jsx

```diff
@@ -28,13 +29,49 @@ import FormattedText from "../components/FormattedText.jsx";
 
 // GUI for displaying answers
 function Question (props) {
-  let { classes, children, questionDefinition, existingAnswer, isEdit, pageActive, preventDefaultView, defaultDisplayFormatter } = props;
-  let { text, compact, description, disableInstructions } = { ...questionDefinition, ...props }
+  let { classes, children, questionDefinition, existingAnswer, answers, isEdit, pageActive, preventDefaultView, defaultDisplayFormatter } = props;
```

Comment:
This looks like a leftover change.
```suggestion
  let { classes, children, questionDefinition, existingAnswer, isEdit, pageActive, preventDefaultView, defaultDisplayFormatter } = props;
```